template/src/components/views/NotFound.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  AppBar,
+  Button,
+  Container,
+  CssBaseline,
+  Grid,
+  Link,
+  Tab,
+  Tabs,
+  Toolbar,
+  Typography,
+} from '@material-ui/core';
+import { NavLink } from 'react-router-dom';
+import UserMenu from './UserMenu';
+import background404 from 'assets/img/404.svg';
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
```

Comment:
Do we need this? Can we reduce these styles with material components?

Reply:
Background image not, the rest'll be removed in next PR